centos_ci.sh

```diff
@@ -308,6 +308,14 @@ function prepare_for_proxy() {
   firewall-cmd --zone=public --add-port=$INTEGRATION_PROXY_CUSTOM_PORT/tcp;
 }
 
+function perform_nightly() {
+  setup_kvm_docker_machine_driver;
+  cd $GOPATH/src/github.com/minishift/minishift
+  make prerelease synopsis_docs link_check_docs
+  MINISHIFT_VM_DRIVER=kvm make integration_all
```

Comment:
@agajdosi , I hope this is the correct check for nightly build.

Reply:
@budhrg yes,it should be. integration_all will run all the feature file including addon-xpaas.feature except the coolstore feature file.

Reply:
@amitkrout , I think we want to run coolstore as well. So is this what I need to pass ?

```
MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS="-tags coolstore"
```

Reply:
@budhrg No, this would just run the coolstore. Try empty GODOG_OPTS. Something like:

    MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS=""

Reply:
@budhrg No. This is not the way to run all the feature including coolstore. 

> MINISHIFT_VM_DRIVER=kvm make integration_all GODOG_OPTS="-tags coolstore"

This will run only coolstore feature. AFAIU this is not the requirement here. I would suggest use ```MINISHIFT_VM_DRIVER=kvm make integration_all``` to run all the feature file except coolstore as there is an open issue on coolstore repo https://github.com/jbossdemocentral/coolstore-microservice/. See https://github.com/jbossdemocentral/coolstore-microservice/issues/90 for more details.

Reply:
we will run ALL tests in nightl, including coolstore